sentinel/sentinel.go

```diff
@@ -59,19 +47,34 @@ func (p Proxy) handleRequestAndRedirect(w http.ResponseWriter, r *http.Request)
 	r.URL.RawQuery = values.Encode()
 
 	parts := strings.Split(r.URL.Path, "/")
-	host := parts[1]
+	target := parts[1]
 	parts = append(parts[:1], parts[1+1:]...)
 	r.URL.Path = strings.Join(parts, "/")
 
-	switch host { // nolint
+	templateURL := *r.URL
+	switch target { // nolint
 	case "btc-mainnet-fullnode":
-		// add username/password to request
-		host = fmt.Sprintf("thorchain:password@%s:8332", host)
+		templateURL.Host = os.Getenv("BTC_FULLNODE_HOST")
```

Comment:
We usually read environment variables when app start up, after that it doesn't read it again.
1) it is more efficient
2) don't really want to allow user change the app's behaviour by just set ENV 
What you reckon?

Reply:
agree it might be good to read these once at startup vs on every request.

Reply:
agreed, good catch